force-app/main/educationCloudSettings/lwc/edcSettingsProductCard/edcSettingsProductCard.js

```diff
@@ -8,21 +8,31 @@ import stgBtnDocumentation from "@salesforce/label/c.stgBtnDocumentation";
 import stgBtnDocumentationActionA11y from "@salesforce/label/c.stgBtnDocumentationActionA11y";
 import stgBtnTrailhead from "@salesforce/label/c.stgBtnTrailhead";
 import stgBtnTrailheadActionA11y from "@salesforce/label/c.stgBtnTrailheadActionA11y";
-export default class EdcSettingsCard extends NavigationMixin(LightningElement) {
+
+import getEDCSettingsProductVModel from "@salesforce/apex/EDCSettingsProductCardController.getEDCSettingsProductVModel";
+
+export default class EdcSettingsProductCard extends NavigationMixin(LightningElement) {
+
+    @api isDisplayProduct;
+    @api productRegistry;
+    
+    @track
+    showThisProduct = false;
```

Comment:
Odd nitpick. Why is this variable across two lines?

Reply:
Done. Thanks